compiler/compiler/pom.xml

```diff
@@ -121,8 +121,19 @@
         <filtering>true</filtering>
       </resource>
     </resources>
-
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>2.3.2</version>
```

Comment:
@dkimitsa, why  so ancient maven-compiler-plugin?

Reply:
there is lot of deprecated code, we hadn't touch it yet as long as it keep working